packages/client/src/sound-board/service/soundboard-service.ts

```diff
@@ -9,6 +10,9 @@ async function queueXSound(name: string) {
   });
 }
 
-export {
-  queueXSound,
-};
+function playXSoundLocally(url: string) {
+  AudioClient.src = url;
+  AudioClient.play();
```

Comment:
Shouldn't this technically be called inside a `load` event?
I think that there is a possibility for `.play()` to be called before the sound is downloaded.